stable/superset/README.md

```diff
@@ -0,0 +1,92 @@
+#  Apache superset
+
+## Introduction
+
+This chart bootstraps an [Apache superset](https://superset.incubator.apache.org/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## TL;DR;
+
+```bash
+$ helm install stable/superset
+```
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure (Only when persisting data)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/superset
+```
+
+The command deploys Superset on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+| Parameter                  | Description                                     | Default                                                      |
+| -----------------------    | ---------------------------------------------   | ------------------------------------------------------------ |
+| `image.repository`                    | `superset` image repository                     | `amancevice/superset`                                                    |
```

Comment:
Please format the table properly.

Reply:
not sure what exactly a proper format means, because the table already rendered correctly. My interpretatation was to make mark down code more readable.